hasura/metadata/actions.graphql

```diff
@@ -65,3 +69,7 @@ type update_profile_response {
   id: Int!
 }
 
+type logout_response {
```

Comment:
instead of the dupe responses we have now for various x_response that just return id, maybe it would be nice to have an id_response type

Reply:
i think thats a good idea